### DISCUSSION TITLE
Planning to work on supporting embedding typst math in milkdown

### DISCUSSION THREAD

**Discussion by quank123wip** [Milkdown/milkdown#1277] [Ideas]
Since academics' enthuiasm on typst is growing these days, it's a good idea to support such feature in a modern editor.

I'm planning to work on this feature months later, and posting this can help me know how many people like this feature! :)

As far as I know, implement this is very complicated by now.Typst community have no official or reliable web renderer support, and I haven't found anyone made a remark plugin support this feature. So all of this work should be built on scratch and wouldn't come to reality very soon. It's important to let me know how many you fellow like this, as it gets me motivated.

**Comment by Saul-Mirone** [comment]
It's an interesting idea. As far as I know, there's no one working on it.
